src/components/Post/Card.tsx

```diff
@@ -19,10 +21,13 @@ export const postCardTestIds = {
   date: 'post-card__date',
 };
 
-const PostCard = ({ post }: PostCardProps) => {
+const PostCard = ({ post, itemsRowWise = false }: PostCardProps) => {
   return (
     <div
-      className="card bg-base-200 shadow-xl my-8 rounded-lg"
+      className={classNames(
+        { 'w-[48%]': itemsRowWise },
```

Comment:
@tyrro Should design the screen for post list with `display: grid`?
Then we can omit this part out 🙌
Reference: https://css-tricks.com/almanac/properties/g/grid-template-columns/#aa-responsive-and-flexible-card-layout-with-dynamic-number-of-items

![2565-07-01 12 07 38](https://user-images.githubusercontent.com/749672/176827897-6ff21ad6-3c93-4935-b733-dd07b54c4ad9.gif)

Reply:
Thanks for the link!
But I think implementing in `grid` will require a lot more change because the parent is also a flexbox here:
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/39977805/177085699-7c28eac2-acca-4a94-a2ba-deccb510fdb6.png">

In order to keep the implementation simple, I kept it as it is!

Reply:
It's OK to me too 👍